css/_chart.scss

```diff
@@ -2,34 +2,20 @@
 // Line chart
 //
 //A chart used to present complex data. We use <a href="http://c3js.org/gettingstarted.html">C3 charts</a> to get nice features like legends, zoom and period selection.
-//<div class="jsprewrap">
-//	<div ng-click="showjs = !showjs" class="ksscopy">JavaScript</div>
-//	<div ng-show="showjs">
-//		<code><pre>
-//c3.generate({
-//		bindto: '#myChart',
-//		data: {
-//				columns: [
-//					['data1', 30, 200, 100, 400, 150, 250],
-//					['data2', 20, 180, 240, 100, 190, 0],
-//				],
-//				selection: {
-//						enabled: true,
-//				},
-//		},
-//		subchart: {
-//				show: true,
-//		},
-//		zoom: {
-//				enabled: true,
-//		},
-//});
-//		</pre></code>
-//	</div>
-//</div>
 //
 //Markup:
-//	<div class="wfm-chart" id="myChart"></div>
+//	<c3chart bindto-id="chart2">
+//	  <chart-column column-id="data 1"
+//			column-name="Data 1"
+//			column-color="#EF5350"
+//			column-values="30,200,100,400,150,250"
```

Comment:
maybe it could be interesting to extract those numbers in a scope variable to show how to link the html to the controller ?